resources/lib/uzg.py

```diff
@@ -70,41 +71,21 @@ def get_ondertitel(self, whatson_id):
             
         def get_play_url(self, whatson_id):
             ##token aanvragen
-            data = self.__get_data_from_url('http://ida.omroep.nl/npoplayer/i.js')
-            token = re.compile('.token\s*=\s*"(.*?)"', re.DOTALL + re.IGNORECASE).search(str(data)).group(1)
+            data = self.__get_data_from_url('http://ida.omroep.nl/app.php/auth')
+            token = re.search(r'token\":"(.*?)"', data).group(1)
             ##video lokatie aanvragen
-            data = self.__get_data_from_url('http://ida.omroep.nl/odi/?prid='+whatson_id+'&puboptions=adaptive&adaptive=yes&part=1&token='+self.__get_newtoken(token))
+            data = self.__get_data_from_url('http://ida.omroep.nl/app.php/'+whatson_id+'?adaptive&adaptive=yes&part=1&token='+token)
+	    xbmc.log("plugin.video.uzg:: data %s" % data)
```

Comment:
Dit is de officiele manier om logging, maar onnodige logging is niet gewenst. Voor debug voeg altijd de python logging toe en gebruik in uzgtest.py welke in mijn root van de repo staat. De item-code pas ik dan weer aan en zo kan ik de logica los testen zonder xbmc op te starten.

Ik kan nu zien dat de pai dus niet meer werkt zonder jouw aanpassing:
`/assets/002/221/545.jpg', 'nebo_id': u'AT_2037144', 'label': u'Zappsport'}, {'thumbnail': u'http://mediadb.omroep.nl/assets/002/346/810.jpg', 'nebo_id': u'VPWON_1250334', 'label': u'Zondag met Lubach'}, {'thumbnail': u'http://mediadb.omroep.nl/assets/002/922/660.jpg', 'nebo_id': u'AT_2061834', 'label': u'Zorg.nu'}]



[{'date': '28-02-2017', 'whatson_id': u'AT_2068366', 'thumbnail': u'http://mediadb.omroep.nl/assets/003/293/508.jpg', 'label': u'(21:25:00) - Zorg.nu'}, {'date': '21-02-2017', 'whatson_id': u'AT_2068365', 'thumbnail': u'http://mediadb.omroep.nl/assets/003/286/952.jpg', 'label': u'(21:25:00) - Zorg.nu'}, {'date': '16-02-2017', 'whatson_id': u'AT_2073582', 'thumbnail': u'http://mediadb.omroep.nl/assets/003/278/352.jpg', 'label': u'(14:20:00) - Zorg.nu'}]




Traceback (most recent call last):
  File "C:\Code\prive\plugin.video.uzg\uzgtest.py", line 13, in <module>
    print uzg.get_play_url('AT_2068366')
  File "C:\Code\prive\plugin.video.uzg\resources\lib\uzg.py", line 73, in get_play_url
    data = self.__get_data_from_url('http://ida.omroep.nl/npoplayer/i.js')
  File "C:\Code\prive\plugin.video.uzg\resources\lib\uzg.py", line 63, in __get_data_from_url
    response = urllib2.urlopen(req)
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 435, in open
    response = meth(req, response)
  File "C:\Python27\lib\urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Python27\lib\urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 410: Gone`